AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -37,8 +37,8 @@ rp5.by,rp5.kz,rp5.ru,rp5.ua,rp5.co.uk#$?##content > div > div[id]:has(> ins.adsb
 livejournal.com#%#//scriptlet("set-constant", "Object.prototype.Adf", "undefined")
 !
 ! rambler
-mail.rambler.ru#%#//scriptlet('abort-on-property-read', 'Object.prototype.forceCallAd.banner')
 !+ NOT_PLATFORM (windows, mac, android)
+rambler.ru#%#//scriptlet("set-constant", "Object.prototype.getPrefixes", "undefined")
 ! passion.ru,eda.ru,quto.ru,rns.online,championat.com,gazeta.ru,lenta.ru,rambler.ru#%#//scriptlet('abort-on-property-read', 'Object.prototype.banner')
```

Comment:
what about their side projects?

Reply:
It also blocks direct on their side projects, but without scriptlet I can't reproduce direct on `eda.ru`